app/tests/test_browse_transferring_body.py

```diff
@@ -0,0 +1,136 @@
+from unittest.mock import patch
+
+from bs4 import BeautifulSoup
+from flask.testing import FlaskClient
+from sqlalchemy import exc
+
+from app.main.db.queries import browse_view_transferring_body
+from app.tests.mock_database import create_two_test_records
+
+
+@patch(
```

Comment:
any differences in these tests will have to merged in with the tests for the normal view when @colinbowen does his integration

Reply:
yes, agreed, it will change once the integration has been performed